source/API_Reference/Event_Webhook/event.md

```diff
@@ -0,0 +1,1043 @@
+
+{% codeblock lang:json %}
+{
+  "unique_args": {
+    "userid": "1123",
+    "template": "welcome"
+  }
+}
+{% endcodeblock %}
+
+You will receive the same unique argument included with the data returned to the Event Webhook:
+
+{% codeblock lang:json %}
+[
+  {
+    "sg_message_id":"sendgrid_internal_message_id",
+    "email": "john.doe@sendgrid.com",
+    "timestamp": 1337966815,
+    "event": "click",
+    "url": "https://sendgrid.com",
+    "userid": "1123",
+    "template": "welcome"
+  }
+]
+{% endcodeblock %}
+
+{% warning %}
+You can create unique arguments with the same keys as reserved words, such as "event" or "email". In this case, SendGrid will default to the reserved key and NOT your unique argument. An example of this is below.
+{% endwarning %}
+
+{% anchor h4 %}
+Reserved Keys in Unique Arguments
+{% endanchor %}
+
+{% codeblock lang:json %}
+//for this example, assume we're sending to john.doe@sendgrid.com
+{
+  "unique_args": {
+    "customerAccountNumber": "55555",
+    "activationAttempt": "1",
+    "New Argument 1": "New Value 1",
+    "email": "jane.doe@sendgrid.com",
+    "event": "SendEmail"
+  }
+}
+{% endcodeblock %}
+
+{% anchor h4 %}
+The resulting webhook call
+{% endanchor %}
+
+{% codeblock lang:json %}
+[
+  {
+    "event": "Processed",
+    "timestamp":"123456789",
+    "customerAccountNumber": "55555",
+    "activationAttempt": "1",
+    "New Argument 1": "New Value 1",
+    "email": "john.doe@sendgrid.com"
+  }
+]
+{% endcodeblock %}
+
+*Note: The event and the email do not get overwritten because "event" and "email" are reserved, and SendGrid defaults to the reserved key for webhooks.*
+
+{% anchor h3 %}
+Custom Arguments
+{% endanchor %}
+
+Any custom arguments that you include with an email sent through [v3 Mail Send]({{root_url}}/API_Reference/Web_API_v3/Mail/index.html) gets added to your Event Webhook response.
+
+For example, if you were to include the following custom arguments in a personalization in your payload to the v3 Mail Send endpoint:
+
+{% codeblock lang:json %}
+{
+  "personalizations": [
+    {
+      "to": [
+        {
+          "email": "example@example.com"
+        }
+      ],
+      "subject": "Hello, World!",
+      "custom_args": {
+        "userid": "1123"
+      }
+    }
+  ],
+  "from": {
+    "email": "from_address@example.com"
+  },
+  "content": [
+    {
+      "type": "text/plain",
+      "value": "Hello, World!"
+    }
+  ]
+}
+{% endcodeblock %}
+
+The Event Webhook response:
+
+{% codeblock lang:json %}
+[
+  {
+    "userid": "1123"
+  }
+]
+{% endcodeblock %}
+
+- <a name="marketingcampaignid"></a>`marketing_campaign_id`
+- <a name="marketingcampaignname"></a>`marketing_campaign_name`
+
+For emails sent through our Marketing Campaigns feature, we will add Marketing Campaigns specific params to the Event Webhook. Both `marketing_campaign_name` and `marketing_campaign_id` are displayed as unique arguments in the event data.
```

Comment:
For emails sent through our Marketing Campaigns feature, we ~~will~~ add Marketing Campaigns specific params to the Event Webhook.